features/object-view-box.yml

```
@@ -0,0 +1,5 @@
+name: object-view-box
+description: The `object-view-box` CSS property defines the subset within an image that should be drawn in the content box of the image.
```

I found "subset … drawn in the context box" to be pretty abstract. What do you think of something like this, that gets at the result more than the method?

```suggestion
description: The `object-view-box` CSS property crops and zooms to an inset area of an image.
```